tests/core/arch-x86/stalker-x86-fixture.c

```diff
@@ -109,12 +109,12 @@ test_stalker_fixture_dup_code (TestStalkerFixture * fixture,
 {
   GumAddressSpec spec;
 
-  spec.near_address = gum_stalker_follow_me;
+  spec.near_address = (gpointer) gum_stalker_follow_me;
```

Comment:
Why are these casts needed (this one and the ones below); does this file get included by C++ code?